src/components/BoundPiecesList/BoundPiecesList.test.js

```diff
@@ -0,0 +1,113 @@
+import { screen } from '@folio/jest-config-stripes/testing-library/react';
+import userEvent from '@folio/jest-config-stripes/testing-library/user-event';
+import { ConfirmationModal } from '@folio/stripes/components';
+
+import '@folio/stripes-acq-components/test/jest/__mock__';
+
+import { renderWithIntl } from '../../../test/jest/helpers';
+import {
+  useBoundPieces,
+  usePiecesMutation,
+} from '../../common/hooks';
+
+import BoundPiecesList from './BoundPiecesList';
+
+jest.mock('@folio/stripes/components', () => ({
+  ...jest.requireActual('@folio/stripes/components'),
+  ConfirmationModal: jest.fn(),
+  TextLink: jest.fn().mockImplementation(({ children, to }) => <a href={to} data-testid="textLink">{children}</a>),
+}));
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  useStripes: jest.fn().mockReturnValue({ hasPerm: jest.fn().mockReturnValue(true) }),
+}));
+jest.mock('../../common/hooks', () => ({
+  useBoundPieces: jest.fn(),
+  usePiecesMutation: jest.fn(),
+}));
+
+const boundPieces = [{
+  isBound: true,
+  displaySummary: 'Electronic item',
+  status: { name: 'Available' },
+  itemId: 'itemId',
+  id: 'id',
+}];
+
+const mockUpdatePiece = jest.fn().mockResolvedValue(() => Promise.resolve());
+
+const renderBoundPiecesList = (props = {}) => renderWithIntl(
```

Comment:
pass also `translationsProperties` as a second arg and then you can check for a specific translation.